scripts/r.import/r.import.py

```diff
@@ -206,6 +217,25 @@ def main():
                 _("Input <%s> successfully imported without reprojection")
                 % GDALdatasource
             )
+            if flags["e"]:
+                try:
+                    gs.message(_("Calculating estimted resolution..."))
+                    raster_info = gs.raster_info(output)
+                    if raster_info:
+                        estres = (raster_info["ewres"] + raster_info["nsres"])/2.0
```

Comment:
To be consistent with the original code,
```suggestion
                        estres = math.sqrt(raster_info["nsres"] * raster_info["ewres"])
```